source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -550,52 +541,11 @@ AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
       swift_runtime->DoArchetypeBindingForType(*stack_frame,
                                                imported_self_type);
```

Comment:
Since we overwrite `imported_self_type`, might be a good thing to assert that the archtetype binding succeeded (`assert(imported_type_type.isValid());`)